tools/docker/builder/rdkb/toolchain/Dockerfile

```
@@ -8,7 +8,7 @@
 FROM ubuntu:xenial
 
 # Update Software repository
-RUN apt-get update
+RUN apt-get update --fix-missing &&
```

```suggestion
RUN apt-get update --fix-missing
```

What occasions? Never experienced an apt-get failure that required this...